rust/Cargo.toml

```diff
@@ -9,3 +9,31 @@ inc = { path = "inc", version = "0.1.0" }
 
 [lib]
 crate-type = ["staticlib"]
+
+[profile.rust-dev]
+inherits = "dev"
+opt-level = 2
+debug = false
+overflow-checks = false
+debug-assertions = false
+strip = "symbols"
+
+[profile.rust-debug]
+inherits = "dev"
+opt-level = 1
+overflow-checks = false
+incremental = false
+
+[profile.rust-sanitize]
+inherits = "dev"
+opt-level = 2
```

Comment:
The `sanitize` mode compiles cpp files using `-Os`, so maybe 's' would be more appropriate here?

Reply:
I expect that both clang and rustc use similar optimization passes for the `s` optimization level considering they both use llvm, but I didn't find any direct comparison that confirms it. There is a slight difference in how they are described in their official documentations though: clang describes `-Os` as "Like `-O2` with extra optimizations to reduce code size", while rust describes it as just: "optimize for binary size".
The description of the `Sanitize` mode provided by scylla is "a mode with optimizations and sanitizers enabled, used for finding memory errors", so I used the mode that is confirmed to at least have some optimizations.

On the other hand, I haven't confirmed that the `-O2` mode in rust is that similar to clang's `-O2`, so for the sake of consistency, maybe the `s` option would be best after all

What do you think?

Reply:
I think that the main purpose of `-Os` is to "optimize for size", so IMO we should use `s` here.

Reply:
OK